### DISCUSSION TITLE
Solar Panel Aligner shows inaccurate information

### DISCUSSION THREAD

**Discussion by CloneWith** [kylecorry31/Trail-Sense#2196] [Q&A] [answered]
this tool works perfectly during the day. But when after sunset, the maximum value kept showing zero, even if the charge time is set to more than night time (for example, it's 18:00 now, and charge for 20h).

![Screenshot_2024-02-04-18-59-21-676_com.kylecorry.trail_sense.png](https://github.com/kylecorry31/Trail-Sense/assets/110881926/e3bb7ee0-5ab6-4268-b00c-137abcef92f2)

may work better with sunrise and sunset infomation support

**Comment by kylecorry31** [comment]
Thank you for letting me know. I don't exactly remember why, but I had intentionally clamped it to the end of the day which is why it doesn't work after dark. I'll take a look and remove that restriction.

**Comment by kylecorry31** [comment, marked as answer]
I've created this issue to track further progress on the fix: https://github.com/kylecorry31/Trail-Sense/issues/2197
